code/src/java/plugin/encounter/EncounterPlugin.java

```diff
@@ -24,7 +24,7 @@
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
 import java.io.File;
-import java.io.FileNotFoundException;
+// import java.io.FileNotFoundException;
```

Comment:
same here

Reply:
One of the tests flagged it as unused. I didn't want to just delete it. so I commented it.

Reply:
Looking back I see 2 PMD rule violations were found by travis, thus it failed.  Commenting these fixed it.

Reply:
In both cases I see I had in a prior commit removed an @throw when there was no throw to be found. Thus the imports became unused and travis complained.

Reply:
Just remove them in that case :-)